h2o-r/h2o-package/R/models.R

```diff
@@ -853,12 +856,19 @@ h2o.staged_predict_proba <- staged_predict_proba.H2OModel
 #' h2o.predict_contributions(prostate_gbm, prostate)
 #' }
 #' @export
-predict_contributions.H2OModel <- function(object, newdata, ...) {
+predict_contributions.H2OModel <- function(object, newdata, output_format = c("Original", "Compact"), ...) {
     if (missing(newdata)) {
         stop("predictions with a missing `newdata` argument is not implemented yet")
     }
+    params <- list(predict_contributions = TRUE)
+    if (!missing(output_format)) {
+        if (!(output_format %in% c("Original", "Compact"))) {
+          stop("type must be one of: Original, Compact")
+        }
+        params$predict_contributions_output_format <- output_format
```

Comment:
Wouldn't it be better to use:
```r
params$predict_contributions_output_format <- match.arg(output_format)
```

Reply:
or something like `match.arg(arg=tolower(output_format), choices = eval(formals()$output_format))` to keep it case insensitive (assuming the vector in declaration is in lowercase).

Reply:
good idea, thanks - I don't think we need to worry about case insensitivity - other places we just use match.arg